hedera-mirror-rest/controllers/networkController.js

```diff
@@ -42,7 +41,7 @@ const networkNodesDefaultSize = 10;
 const networkNodesMaxSize = 25;
 
 class NetworkController extends BaseController {
-  static contentTypeTextPlain = 'text/plain; charset=utf-8';
+  static contentTypeTextPlain = {'Content-type' : 'text/plain; charset=utf-8'};
```

Comment:
shouldn't this be all lower case and even better use a constant for the key instead?

```suggestion
  static contentTypeTextPlain = {'content-type' : 'text/plain; charset=utf-8'};
```